docs/nodejs/README.md

```diff
@@ -179,18 +179,12 @@ Installing application metrics scripts
 
       found 0 vulnerabilities
 
-  - Done (3.0s)
+  - Done (2.6s)
 - Running scripts
   - No build scripts found
 - Configuring default processes
   - Skipping default web process (Procfile detected)
-
-! Warning: `node_modules` checked into source control
```

Comment:
@colincasey I've just noticed the `node_modules` warning used to be shown here. This will be because these tutorials `npm install` the getting started guide locally and then perform a `pack build` locally too. And `pack build` will use whatever in the working directory rather than only what is git committed. 

Should the Node.js getting started guide have an `io.buildpacks.exclude` entry in `project.toml` (the CNB equivalent of a `.dockerignore`) to exclude `node_modules`?
https://buildpacks.io/docs/reference/config/project-descriptor/#iobuildpacks-_table-optional_